app/xl_NewFileByTemplate.py

```diff
@@ -73,7 +73,7 @@ def SetUp(xlApp):
 			ws = xlApp.Sheets(tempSheetName)
 			ws.Activate
 	        
-			xlApp.ActiveWorkbook.SaveAs(newFilePath + "\\" + str(newFileName[i]) + ".xlsx")
+			xlApp.ActiveWorkbook.SaveAs(newFilePath + "\\" + str(newFileName) + ".xlsx")
 			xlApp.ActiveWorkbook.Close(False)
 			xlApp.screenUpdating = True
 			Marshal.ReleaseComObject(ws)
```

Comment:
Ah ok, i haven't thought of that case. Could we add a condition then to distinguish between single file and list of files? Would make it more dummy proof :)